dashboards-observability/public/components/notebooks/components/__tests__/note_table.test.tsx

```diff
@@ -81,4 +81,39 @@ describe('<NoteTable /> spec', () => {
     utils.getByText('Actions').click();
     utils.getByText('Rename').click();
   });
-});
+
+  it('create notebook', () => {
```

Comment:
Do we need to add or modify any cypress tests?

Reply:
Good call out let me check

Reply:
@mengweieric fixed in: [3dc8296](https://github.com/opensearch-project/dashboards-observability/pull/1308/commits/3dc8296418b01d3cdd8ea7054db6b67a67287336). Do we want to run cypress test as a part of github action? It's a bit long but we should know about it somehow/they should be updated on any changes or flakiness or they should be removed in my opinion - why do we have tests that don't run on a regular basis?

Reply:
Thanks for the changes, yes that's one task I currently have on plate, just need to resolve all the failing tests which may need some helping hands. Not sure if we should simply just add it to run on every PR or something like a nightly run.

Reply:
@mengweieric  I vote for nightly run 👍🏽 😄